### PR TITLE
Add Disco performance to README

### PR DESCRIPTION
**Pull request by LeshengJin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ LLM Performance Benchmarking
 
 ## Performance
 
+All experiments are based on fp16 activation and compute, decoding 256 tokens with a prompt "What is the meaning of life?". And all numbers are based on PCIe, not NVLink.
+
 ### Int4-quantized, Single GPU
```
Comment (junrushao) [inline README.md:8]: Change all int4 to 4-bit

File: README.md [hunk 2]
```diff
@@ -12,11 +14,43 @@ LLM Performance Benchmarking
 | Llama2-7B  | RTX 4090    | 204.8             | 177.46               | 151.1               |
 | Llama2-13B | RTX 4090    | 113.5             | 105.94                | 88.0                |
 
-All experiments are based on int4-quantized weights, fp16 activation and compute, decoding for 256 tokens with a prompt "What is the meaning of life?".
-
 ### FP16, Multi-GPU
 
-TBD
+| Model         | GPU      | MLC LLM (tok/sec) | Exllama V2 (tok/sec) | Llama.cpp (tok/sec) | vLLM (tok/sec) |
```
Comment (junrushao) [inline README.md:19]: Add a note saying Exllama doesn't support fp16

File: README.md [hunk 3]
```diff
@@ -12,11 +14,43 @@ LLM Performance Benchmarking
 | Llama2-7B  | RTX 4090    | 204.8             | 177.46               | 151.1               |
 | Llama2-13B | RTX 4090    | 113.5             | 105.94                | 88.0                |
 
-All experiments are based on int4-quantized weights, fp16 activation and compute, decoding for 256 tokens with a prompt "What is the meaning of life?".
-
 ### FP16, Multi-GPU
```
Comment (junrushao) [inline README.md:17]: Multi-GPU, NVIDIA

File: README.md [hunk 4]
```diff
@@ -12,11 +14,43 @@ LLM Performance Benchmarking
...
+|               | A100 x 8 | 59.4    | 32.23   | 11.2      | 17.6  |
+| Llama2-70B    | A10G x 2 | 19.8    | 13.48   | 11.98     | 12.89 |
+|               | A10G x 4 | 34.3    | 13.48   | 13.37     | 16.91 |
+|               | A10G x 8 | 47.7    | 13.48   | 8.01      | 20.79 |
+| CodeLlama-34B | A10G x 1 | 23.4    | 25.81   | 22.72     | 19.76 |
+|               | A10G x 2 | 38.4    | 25.86   | 21.93     | 23.67 |
+|               | A10G x 4 | 61.2    | 25.84   | 23.53     | 29.83 |
+|               | A10G x 8 | 84.2    | 25.82   | 13.25     | N/A   |
+
+#### Game GPUs
```
Comment (junrushao) [inline README.md]: 4bit, Multi-GPU, AMD

File: README.md [hunk 5]
```diff
@@ -3,6 +3,8 @@ LLM Performance Benchmarking
 
 ## Performance
```
Comment (junrushao) [inline README.md:5]: Let's have the following sections:
* Single GPU, 4-bit
* Multiple NVIDIA GPU, fp16
* Multiple AMD GPU, 4-bit

Comment (LeshengJin) [reply README.md:5]: Are we supposed to include Multiple NVIDIA GPU, 4-bit?

**Review by junrushao — APPROVED** [review]

**Pull request merged**